### DISCUSSION TITLE
I wanted to inquire if there are any projects within our community that involve Rust programming language.

### DISCUSSION THREAD

**Discussion by bharattech** [edgexfoundry/community#167] [Q&A] [answered]
I wanted to inquire if there are any projects within our community that involve Rust programming language.
Rust has gained popularity for its focus on memory safety, performance, and concurrency. 

If anyone is aware of or involved in Rust-related projects, kindly share the details or provide a brief overview. Your insights will help us assess the feasibility of participating and making meaningful contributions.

**Comment by lenny-goodell** [comment, marked as answer]
Nope, EdgeX Foundry uses Go as the primary language. There is a C Device SDK and a few C Device Services.
